packages/toolpad-components/src/Tabs.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import { Tabs as MUITabs, Tab } from '@mui/material';
+import { createComponent } from '@mui/toolpad-core';
+
+interface TabProps {
+  title: string;
+  name: string;
+}
+
+interface Props {
+  active: string;
+  onChange: (value: number) => void;
+  tabs: TabProps[];
+  defaultValue: string;
+}
+
+function Tabs({ active, onChange, tabs, defaultValue }: Props) {
+  return (
+    <MUITabs
+      value={active || defaultValue}
+      onChange={(event, value) => {
+        onChange(value);
+      }}
+    >
+      {tabs.map(({ title, name }) => (
+        <Tab label={title} value={name} key={name} />
+      ))}
+    </MUITabs>
+  );
+}
+
+export default createComponent(Tabs, {
+  layoutDirection: 'horizontal',
+  argTypes: {
+    active: {
```

Comment:
This should be `value`

Reply:
Can you clarify what you mean? the name of prop? if yes, why?

Reply:
> the name of prop? if yes, why?

yes, because:

1. Conventions: we generally use the combination `value`/`onChange` to denote the main controlled prop of a component (yes, there are specific situations where we deviate from this, e.g. dataGrid.)
2. The MUI Tabs uses `value`
3. This component as it currently is, is in essence a Select, and the MUI Select uses `value`
4. The word "active" is an adjective, and a "tab" is "a thing", so we should use a noun for naming it in our code. the word "value" is a noun.